snapcraft/extractors/appstream.py

```diff
@@ -138,7 +140,8 @@ def _get_xslt():
 def _get_value_from_xml_element(tree, key) -> Optional[str]:
     node = tree.find(key)
     if node is not None and node.text:
-        return node.text.strip()
+        # TODO: do this in the xslt
```

Comment:
Maybe ditch this comment if not going to happen (and perhaps just add why it's stripping all lines)

Reply:
It is, I just need to learn how :-)

Reply:
Added a extended comment